setup.py

```diff
@@ -39,6 +39,7 @@
         'taca.subcommands': [
             'storage = taca.storage.cli:storage',
             'analysis = taca.analysis.cli:analysis',
+            'bioinfo = taca.utils.cli:bioinfo',
```

Comment:
welp, just renamed that delivery. I guess I'll have to find something else
then. Any suggestions ?

On Thu, Jul 9, 2015 at 10:34 AM, Pontus Larsson <notifications@github.com>
wrote:

> In setup.py
> <https://github.com/SciLifeLab/TACA/pull/134#discussion_r34233765>:
>
> > @@ -39,6 +39,7 @@
> >          'taca.subcommands': [
> >              'storage = taca.storage.cli:storage',
> >              'analysis = taca.analysis.cli:analysis',
> > +            'bioinfo = taca.utils.cli:bioinfo',
>
> Could be good to bear in mind that there is already the ’deliver’
> subcommand to TACA (with the taca-ngi-pipeline plugin), so avoid naming the
> subcommand too similar. 9 jul 2015 kl. 07:59 skrev Francesco <
> notifications@github.com<mailto:notifications@github.com>>: In setup.py<
> https://github.com/SciLifeLab/TACA/pull/134#discussion_r34225661>:
>  @@ -39,6 +39,7 @@ 'taca.subcommands': [ 'storage =
> taca.storage.cli:storage', 'analysis = taca.analysis.cli:analysis', +
> 'bioinfo = taca.utils.cli:bioinfo',
> @Galithil <https://github.com/Galithil><https://github.com/Galithil> this
> subcommands are used for the delivery page, so maybe better use a more
> intuitive name than 'bioinfo'. What about 'deliveries' or 'delivery-page' —
> Reply to this email directly or view it on GitHub<
> https://github.com/SciLifeLab/TACA/pull/134/files#r34225661>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SciLifeLab/TACA/pull/134/files#r34233765>.
>